processor/src/range/mod.rs

```diff
@@ -41,39 +24,27 @@ pub struct RangeCheckTraceTable {
 /// into 16-bits, but rather keeps track of all 16-bit range checks performed by the VM.
 ///
 /// ## Execution trace
-/// Execution trace generated by the range checker consists of 4 columns. Conceptually, the table
-/// is split into two sets of rows (two segments): an 8-bit segment and a 16-bit segment. The
-/// 8-bit segment must enumerate all 256 possible 8-bit values. Values must be in increasing order,
-/// but duplicates are allowed. The 16-bit segment must start with value 0 and end with value
-/// 65535. The values must also be in increasing order but can be up to 255 values apart, and
-/// duplicates are also allowed.
+/// Execution trace generated by the range checker consists of 3 columns. Conceptually, the table
+/// starts with value 0 and end with value 65535. The values must be in increasing order and the
+/// jump allowed between two values should be a power of 3 less than 7, and duplicates are allowed.
```

Comment:
nit: everything after the first sentence, I'd move into the bullet point about column `v` below the diagram.